test/hack/bin/check-opsrecipes.sh

```diff
@@ -27,15 +28,39 @@ isInArray () {
     return 1
 }
 
+# merge_docs () merges a Hugo docs hierarchy from a source directory (first arg) into a destination directory (second arg).
+merge_docs() {
+    if [ ! -d "$1/content/docs/." ] ; then
```

Comment:
These are `test` conditions as supported by `sh`.
In other places in this script we use the `bash` syntay `[[ ... ]]`.
Could we keep it consistent and use the same all around?

Reply:
```suggestion
    if [[ ! -d "$1/content/docs/." ]] ; then
```